src/main/java/uk/gov/hmcts/reform/sandl/snlevents/model/response/HearingWithSessionsResponse.java

```diff
@@ -48,5 +51,12 @@ public HearingWithSessionsResponse(Hearing hearing) {
             .map(hp -> new ViewSessionResponse(hp.getSession()))
             .sorted(comparing(ViewSessionResponse::getStart))
             .collect(Collectors.toList());
+        this.hearingPartsVersions = hearing.getHearingParts().stream().map(hp -> {
```

Comment:
what is this for? why do we need list of versions?

Reply:
Its needed for an optimistic lock. We need to send it with unlist request in order to check that this entity haven't been currently locked by someone else.